### DISCUSSION TITLE
Not able to display Jupyter notebook

### DISCUSSION THREAD

**Discussion by tusharganguli** [alshedivat/al-folio#2870] [Q&A]
- I added a my file to assets/jupyter/ad_timeseries.ipynb. 
- Then I added my markdown file and as instructed included the following code:

```liquid
{::nomarkdown}
{% assign jupyter_path = '/assets/jupyter/ad_timeseries.ipynb' | relative_url %}
{% capture notebook_exists %}{% file_exists /assets/jupyter/ad_timeseries.ipynb %}{% endcapture %}
{% if notebook_exists == 'true' %}
  {% jupyter_notebook jupyter_path %}
{% else %}
  <p>Sorry, the notebook you are looking for does not exist.</p>
{% endif %}
{:/nomarkdown}
```

- When I open the blog page, it keeps displaying the following information:
```
{::nomarkdown}



  <div
  class="jupyter-notebook"
  style="position: relative; width: 100%; margin: 0 auto;">
  <div class="jupyter-notebook-iframe-container">
    <iframe
      src="/assets/jupyter/ad_timeseries.ipynb.html"
      style="position: absolute; top: 0; left: 0; border-style: none;"
      width="100%"
      height="100%"
      onload="this.parentElement.style.paddingBottom = (this.contentWindow.document.documentElement.scrollHeight + 10) + 'px'"></iframe>
  </div>
</div>


{:/nomarkdown}
```

If someone could kindly help as to what am I doing wrong here?

**Comment by george-gca** [comment]
Just checked your page and you seem to have solved this issue.

**Comment by tusharganguli** [comment]
Thanks for the prompt response. Actually the pages that you checked had static html pages embedded, I converted ipynb to html manually and embedded them in the md files. However, today when I tried to create a sample md file with embedded ipynb file to highlight the problem, it worked perfectly. So, it seems it is working now and I have no idea why!

**Reply by george-gca** [reply]
Remember that the build take some time. Maybe it was a matter of waiting for the build to finish.

**Comment by tusharganguli** [comment]
I think I might have an idea of where the problem could be. When I explicitly converted .ipynb to html I was initially getting an error [State Error](https://github.com/jupyter/nbconvert/issues/1731). Possibly the notebooks were not getting converted successfully and once the error was removed it worked fine.
